src/main/kotlin/org/malachite/estella/services/RecruitmentProcessService.kt

```diff
@@ -100,9 +101,23 @@ class RecruitmentProcessService(
     fun updateEndDate(processId: Int, endDate: Date) {
         val userFromJWT = securityService.getHrPartnerFromContext()
         val process = getProcess(processId)
-        if (process.offer.creator.id != userFromJWT?.id) throw UnauthenticatedException()
-        if (process.startDate.after(endDate)) throw InvalidEndDateException()
+        assertCanPerformOperation(process, userFromJWT)
+        if (process.isStarted()) throw ProcessAlreadyStartedException(processId)
```

Comment:
Czyli nie możemy zupdateować końcowej daty procesowi który się zaczął? To trochę kiepskie bo gdyby było mało kandydatów to musielibyśmy po zamknięciu procesu otworzyć go ponownie XD

Reply:
@Rados13 pisałeś o tym na froncie też?